examples/user-config/network/network.yaml

```diff
@@ -0,0 +1,87 @@
+#stack_prefix: default
+
+ec2_vpc:
+  stack_name: test-aem-vpc-stack
+  tag_name: AEM Stack VPC
+  cidr_block: 11.0.0.0/16
+  dhcp_option: dopt-75958817
+
+network:
+  stack_name: aem-network-stack
+
+  internet_gateway:
+    tag_name: AEM VPC Internet Gateway
+
+  publish_dispatcher_elb:
+    subnet_a:
```

Comment:
Also need to discuss whether `subnet_<letter>` could be replaced with the az name, or perhaps removed altogether and relies on array instead.

Reply:
This is resolved in my latest commits.